lib/shutil_custom/__init__.py

```diff
@@ -28,7 +28,7 @@ def special_file(fn):
             if stat.S_ISFIFO(st.st_mode):
                 raise SpecialFileError("`%s` is a named pipe" % fn)
 
-    with os.open(src, READ_FLAGS) as fin, os.open(dst, WRITE_FLAGS) as fout:
+    with open(src, READ_FLAGS) as fin, open(dst, WRITE_FLAGS) as fout:
```

Comment:
The builtin `open` does not support flags, thus the use of `os.open`.

Reply:
Yep and `os.open` doesn't support context managers for python < 3.6(?). What shall we do?

Reply:
You use `os.fdopen` as the context manager with the file descriptors returned by `os.open`